# Standard Message to send commands of angular velocity to the PhidGets
float32 vel
# Variable dir returns the direction of the velocity of the motor, where 1 is clockwise and 
#      1 is clockwise and -1 is anti-clockwise
int32 dir

